src/top.h

```diff
@@ -104,6 +104,9 @@
 #define CONFIG_TOIT_BYTE_DISPLAY 1
 #define CONFIG_TOIT_BIT_DISPLAY 1
 #define CONFIG_TOIT_FONT 1
+#if !defined(TOIT_WINDOWS) and !defined(BUILD_32)
```

Comment:
This should not be determined here.
The CMakeLists.txt should set the constant depending on whether it found the zlib.
```cmake
if (ZLIB_FOUND)
  add_definitions(-DCONFIG_TOIT_FULL_ZLIB)
endif()
```